### DISCUSSION TITLE
연속적인 메시지 전송은 안되나요?

### DISCUSSION THREAD

**Discussion by namsa87** [storycraft/node-kakao#758] [Q&A]
mysql 연동해 가져온 데이터를 channel.sendChat() 으로 오픈채팅방에 전송하려는데 로그는 정상적으로 데이터를 가져오지만
메시지는 2건 전송되고 끝나네요. 혹시 연속적인 메시지 전송방법이 따로 있나요?
그리고 channel.kickUser() 사용방법이 어떻게 될까요?

**Comment by wkswkfkwks** [comment]
나쁜짓 하지마세요

**Reply by namsa87** [reply]
나쁜용도(광고 또는 폭탄메시지 등)가 아니라 같은 톡방 사람들에게 동의하에 정보를 공유하려고 합니다.
(주 내용으론 거주하는 지역 실거래 내역 알림)

**Reply by pupuarw** [reply]
정보 전달인데 도배가 왜필요하나요?

**Reply by benesiaxd** [reply]
ㅋㅋㅋ.. 굳이 2회이상 도배할 필요가있나요

방법 알려드리고 영구정지 먹여드리려다 참습니다.

**Reply by wkswkfkwks** [reply]
그렇군요!!! sendChat()함수를 원하시는 length만큼 루프돌리면 됩니다!!!!!!!! 정보가 여러가지라면 루프문안에서 조건을 거시면 되겠죠~?!

**Reply by namsa87** [reply]
@wkswkfkwks 감사합니다. 선생님 ㅠㅠ

**Reply by lee-jox** [reply]
정보전달도 말풍선 나눠서 올리는것은 큰 차이가 있습니다. 다만 카톡에서 빠르게 올리는것을 감지하니 시간텀을 둬서하시는것이 좋겠네요.

**Comment by ggoek34** [comment]
단순 매크로도 아니고 로코이용해서 폭탄광고보내면 실형떨어집니다 적당히들좀하세요
